src/commands/DownloadCommand.ts

```diff
@@ -22,35 +25,49 @@ export class DownloadCommand {
    * @param {string} outputFolder
    * @memberof DownloadCommand
    */
-  constructor(private originOrganization: string, originApiKey: string, private outputFolder: string) {
-    this.oganization = new Organization(originOrganization, originApiKey);
+  constructor(originOrganization: string, originApiKey: string, private outputFolder: string) {
+    this.organization = new Organization(originOrganization, originApiKey);
   }
 
   /**
    * Downloads the fields of organization. Produces a ${objectName}.json file
    * @memberof DownloadCommand
    */
   downloadFields() {
-    // hack: FieldController needs 2 orgs becaase initially it was  meant for comparison. Feed it.
```

Comment:
@jpdery26 I updated all the Controller's constructors so it can accept only one `organization`.